Solutions/CohesitySecurity/Tests/Alert.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import json
+
```

Comment:
Is there a reason to name the file starting with a capital letter, not in sync with other files in the same directory?

Reply:
done